.test-infra/jenkins/job_CloudMLBenchmarkTests_Python.groovy

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import CommonJobProperties as commonJobProperties
+import PhraseTriggeringPostCommitBuilder
+import CronJobBuilder
+
+def cloudMLJob = { scope ->
+  scope.description('Runs the TFT Criteo Examples on the Dataflow runner.')
+
+  // Set common parameters.
+  commonJobProperties.setTopLevelMainJobProperties(scope)
+
+  // Gradle goals for this job.
+  scope.steps {
+    gradle {
+      rootBuildScriptDir(commonJobProperties.checkoutDir)
+      commonJobProperties.setGradleSwitches(delegate)
+      commonJobProperties.setTopLevelMainJobProperties(delegate, 'master', 360)
```

Comment:
Seed job has a new error:

```
ERROR: (CommonJobProperties.groovy, line 43) No signature of method: javaposse.jobdsl.dsl.helpers.step.GradleContext.properties() is applicable for argument types: (CommonJobProperties$_setTopLevelMainJobProperties_closure1) values: [CommonJobProperties$_setTopLevelMainJobProperties_closure1@21033ade]
```

I think this should be:
```suggestion
      commonJobProperties.setTopLevelMainJobProperties(scope, 'master', 360)
```

Reply:
Ah..I always get confused here. Let me clean this up

Reply:
It should work now. Thanks for running seed jobs.

Reply:
It ran successfully this time - https://ci-beam.apache.org/job/beam_SeedJob/10944